_articles/bluetooth.md

```diff
@@ -13,8 +13,68 @@ section: network-troubleshooting
 
 ---
 
+# Important Notes About Bluetooth
+
+Bluetooth is a bit odd. 
+There are a lot of factors that go into whether Bluetooth devices work as expected.
+
+
+## Bluetooth version
+
+Bluetooth 5.0 is backwards compatible with older Bluetooth versions, but older bluetooth versioned devices are not always compatible with newer devices.
+
+
+## Signal Interference
+
+Bluetooth uses the same bandwidth as the 2.4Ghz Wi-Fi band, and in most of our machines it is on the same chip as the Wi-Fi module. They usually have two antennae, one for Bluetooth, and one for Wi-Fi, but it is possible for other Wi-Fi or Bluetooth devices signals to cross and to cause connection issues.
+
+
+## Device Specific Differences
+
+Every Bluetooth device is different. They use the same or similar protocols, but the printed circuit boards (PCBs), are specific to each device, and the firmware they are running is often custom designed, and closed-source.
+
+Some of the code and technologies that make Bluetooth work reliably are patented, and only device vendors who have licensed the permission to use that patented technology will be able to experience the full benefits (see "audio input/output" section below.)
```

Comment:
I would add that Headphones are a big part of this issue, Sony and Apple have a lot of patented tech.

Reply:
Also, reference them to later in the article when we talk about that sort of tech.

Reply:
Good feedback. Only partially added to avoid bashing/name-calling.